internal/hardware/validator.go

```diff
@@ -244,8 +285,20 @@ func (v *validator) getOCPHostRoleRequirementsForVersion(cluster *common.Cluster
 	return *requirements.WorkerRequirements, nil
 }
 
+func (v *validator) getOCPInfraEnvHostRoleRequirementsForVersion(infraEnv *common.InfraEnv, role models.HostRole) (models.ClusterHostRequirementsDetails, error) {
+	requirements, err := v.getOCPRequirementsForVersion(infraEnv.OpenshiftVersion)
+	if err != nil {
+		return models.ClusterHostRequirementsDetails{}, err
+	}
+
+	if role == models.HostRoleMaster {
+		return *requirements.MasterRequirements, nil
+	}
+	return *requirements.WorkerRequirements, nil
+}
+
 func (v *validator) getPreflightOCPRequirements(cluster *common.Cluster) (*models.HostTypeHardwareRequirementsWrapper, error) {
```

Comment:
Should we add 'Cluster' to the name of the method to be aligned with the other methods?

Reply:
fixed